react/Textarea/Textarea.js

```diff
@@ -71,7 +75,9 @@ export default class Textarea extends Component {
   static defaultProps = {
     className: '',
     description: '',
-    inputProps: {}
+    inputProps: {},
+    initalRows: 15,
```

Comment:
(note there's a typo here as well inital rather than initial)

```suggestion
    initialGridRows: 15,
```